src/styles/blocks/catalog.scss

```diff
@@ -5,12 +5,14 @@
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
+  perspective: 400px;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: transform 300ms;
```

Comment:
You have to add here all properties you change on hover, also move transition-duration (300ms) to variable